File: Beginner/1114.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

/*
 * 
*/
public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		Locale.setDefault(Locale.US);
		int x, y;
		do {

			x = sc.nextInt();
			y = sc.nextInt();

			if (x > 0 && y > 0) {
				System.out.println("primeiro");
			}
			if (x < 0 && y > 0) {
				System.out.println("segundo");
			}
			if (x < 0 && y < 0) {
				System.out.println("terceiro");
			}
			if (x > 0 && y < 0) {
				System.out.println("quarto");
			}
		} while (x != 0 && y != 0);

		sc.close();
	}

}

File: Beginner/1114.py
x = int(input())
y = int(input())

while x != 0 and y != 0:

    if x > 0 and y > 0:
        print("primeiro")
    elif x < 0 and y > 0:
        print("segundo")
    elif x < 0 and y < 0:
        print("terceiro")
    elif x > 0 and y < 0:
        print("quarto")

    x = int(input())
    y = int(input())